cryptography/src/commonMain/kotlin/com/wire/kalium/cryptography/MLSClient.kt

```diff
@@ -8,6 +8,14 @@ typealias MLSKeyPackage = ByteArray
 
 interface MLSClient {
 
+    /**
+     * Release any locks the C code have on the MLS resources and
+     * delete local MLS DB and files
+     *
+     * @return true when delete is successful and false otherwise
+     */
+    fun clearLocalFiles(): Boolean
```

Comment:
An exception instead of true or false at least would provide more information on why it failed.

Reply:
The `deleteRecursively()` used in JVM/Android the returns Boolean, and it throws error only if the path is null, so it will never throw.
I can't say the same about other platforms.